src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -712,8 +715,8 @@ - (void)testConcurrentRPCsWithErrorsWithV2API {
 
     RMTSimpleRequest *request = [RMTSimpleRequest message];
     request.responseType = RMTPayloadType_Compressable;
-    request.responseSize = 314159;
-    request.payload.body = [NSMutableData dataWithLength:271828];
+    request.responseSize = SMALL_PAYLOAD_SIZE;
```

Comment:
Curious if we will reduce any test coverage (e.g. certain behaviors only when payload is above a certain threshold) by reducing the payload size? If so maybe consider choosing a "medium" sized payload? :)

Reply:
large size payload should already be covered in two separate tests here ([testLargeUnaryRPCWithV2API](https://github.com/grpc/grpc/blob/fd3bd70939fb4239639fbd26143ec416366e4157/src/objective-c/tests/InteropTests/InteropTests.m#L660) and [testLargeUnaryRPC](https://github.com/grpc/grpc/blob/fd3bd70939fb4239639fbd26143ec416366e4157/src/objective-c/tests/InteropTests/InteropTests.m#L576)).  I think this one is mainly focus on making sure a short burst of unary requests did finish and closing with various state code.